test/unit/document/Document.spec.js

```diff
@@ -329,36 +328,36 @@ describe('Document', () => {
     });
 
     it('should set binary encoded field directly', () => {
-      const path = 'dataObject.binaryObject.base64Value';
+      const path = 'dataObject.binaryObject.byteArrayProperty';
       const buffer = Buffer.alloc(36);
 
       const result = document.set(path, buffer);
 
       expect(result).to.equal(document);
 
-      const valueToSet = new EncodedBuffer(buffer, 'base64');
+      const valueToSet = new EncodedBuffer(buffer, EncodedBuffer.ENCODING.BASE64);
 
       expect(lodashSetMock).to.have.been.calledOnceWith(document.data, path, valueToSet);
     });
 
     it('should set binary field as part of object', () => {
       const path = 'dataObject.binaryObject';
-      const value = { base64Value: Buffer.alloc(36) };
+      const value = { byteArrayProperty: Buffer.alloc(36) };
 
-      lodashGetMock.returns(value.base64Value);
+      lodashGetMock.returns(value.byteArrayProperty);
 
       const result = document.set(path, value);
 
       expect(result).to.equal(document);
 
-      const valueToSet = new EncodedBuffer(value.base64Value, 'base64');
+      const valueToSet = new EncodedBuffer(value.byteArrayProperty, 'base64');
```

Comment:
new EncodedBuffer(value.byteArrayProperty, EncodedBuffer.ENCODING.BASE64);

Reply:
Let's keep it as it is for now. we will remove EncodedBuffer in the next PR